.github/CODEOWNERS

```diff
@@ -1503,6 +1503,7 @@ packages/kbn-monaco/src/esql @elastic/kibana-esql
 /x-pack/test_serverless/**/test_suites/**/ml/ @elastic/ml-ui
 /x-pack/test_serverless/**/test_suites/common/management/transforms/ @elastic/ml-ui
 /x-pack/test/api_integration/services/ml.ts @elastic/ml-ui
+/x-pack/plugins/security_solution/public/common/components/ml @elastic/ml-ui
```

Comment:
The ML UI team don't own the code in this folder - the components are part of the security solution. This line should just be removed I think as this path is already in place

```
x-pack/plugins/security_solution @elastic/security-solution
```